[2023-07-23T00:41:47.953+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-dag.extract scheduled__2023-07-22T00:00:00+00:00 [queued]>
[2023-07-23T00:41:47.968+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-dag.extract scheduled__2023-07-22T00:00:00+00:00 [queued]>
[2023-07-23T00:41:47.969+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-07-23T00:41:47.987+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): extract> on 2023-07-22 00:00:00+00:00
[2023-07-23T00:41:47.992+0000] {standard_task_runner.py:57} INFO - Started process 4577 to run task
[2023-07-23T00:41:48.008+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my-first-dag', 'extract', 'scheduled__2023-07-22T00:00:00+00:00', '--job-id', '1127', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpi3rby8w8']
[2023-07-23T00:41:48.020+0000] {standard_task_runner.py:85} INFO - Job 1127: Subtask extract
[2023-07-23T00:41:48.102+0000] {task_command.py:410} INFO - Running <TaskInstance: my-first-dag.extract scheduled__2023-07-22T00:00:00+00:00 [running]> on host 70d871733d27
[2023-07-23T00:41:48.228+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ramesh@somemail.com' AIRFLOW_CTX_DAG_OWNER='Ramesh Sannareddy' AIRFLOW_CTX_DAG_ID='my-first-dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-07-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-22T00:00:00+00:00'
[2023-07-23T00:41:48.231+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-23T00:41:48.235+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cut -d":" -f1,3,6 /etc/passwd > /home/project/***/dags/extracted-data.txt']
[2023-07-23T00:41:48.258+0000] {subprocess.py:86} INFO - Output:
[2023-07-23T00:41:48.262+0000] {subprocess.py:93} INFO - /bin/bash: line 1: /home/project/***/dags/extracted-data.txt: No such file or directory
[2023-07-23T00:41:48.263+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-07-23T00:41:48.280+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-07-23T00:41:48.289+0000] {taskinstance.py:1350} INFO - Marking task as UP_FOR_RETRY. dag_id=my-first-dag, task_id=extract, execution_date=20230722T000000, start_date=20230723T004147, end_date=20230723T004148
[2023-07-23T00:41:48.325+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 1127 for task extract (Bash command failed. The command returned a non-zero exit code 1.; 4577)
[2023-07-23T00:41:48.373+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-23T00:41:48.405+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
